Swift.sublime-syntax

```diff
@@ -344,10 +344,10 @@ contexts:
       scope: keyword.operator
     - match: \w+
       scope: support.class
-    - match: '>'
+    - match: '>\s*'
       pop: true
   class_type:
-    - match: \w+
+    - match: \w+\s*
```

Comment:
Are all these whitespace checks necessary?  Sublime's syntax checker already skips preceding & trailing whitespace.  Try removing them and see if everything still works.